AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2172,6 +2188,12 @@ - (void)setInterfaceState:(ASInterfaceState)newState
         }
       }
     }
+    
+    if (nowDisplay) {
```

Comment:
@lappp9 we could collapse this for now like the visible example below; e.g. displayStateDidChange:nowDisplay